website/vercel.json

```diff
@@ -2,21 +2,23 @@
   "headers": [
     {
       "source": "/(.*)",
-      "headers" : [
+      "headers": [
         {
-          "key" : "X-Robots-Tag",
-          "value" : "all"
+          "key": "X-Robots-Tag",
+          "value": "all"
         }
       ]
     },
     {
       "source": "/testing(.*)",
-      "headers" : [
+      "headers": [
         {
-          "key" : "X-Robots-Tag",
-          "value" : "none"
+          "key": "X-Robots-Tag",
+          "value": "none"
         }
       ]
     }
-  ]
+  ],
+  "framework": "ember",
```

Comment:
@alex-ju I could try taking it out and see if it still works, I added it as part of following the docs and afterwards thought that it would at least work as a failure redundancy measure.